packages/automerge-repo-react-hooks/test/useDocuments.test.tsx

```diff
@@ -62,6 +62,18 @@ describe("useDocuments", () => {
     )
   })
 
+  it("returns a collection of loaded documents immediately, given a list of ids", async () => {
```

Comment:
This sort of duplicates the above test. I wasn't sure if this should be its own test or be folded into the above.